pkg/cmd/repo/create/create.go

```diff
@@ -697,9 +680,5 @@ func getVisibility() (string, error) {
 }
 
 func normalizeRepoName(name string) string {
-	re := regexp.MustCompile(`[^a-zA-Z0-9-._]+`)
-	if normalizedRepoName := strings.TrimSuffix(re.ReplaceAllString(name, "-"), ".git"); normalizedRepoName != "" {
-		return normalizedRepoName
-	}
-	return "-"
+	return strings.TrimSuffix(regexp.MustCompile(`[^\w.-]+`).ReplaceAllString(name, "-"), ".git")
```

Comment:
The color switching for this isn't a bad idea, but I think it would be overkill for this feature right now. Let's hold off for now and see whether we feel it's actually necessary in the future.